nucleus/admin/rest/rest-client/src/main/java/org/glassfish/admin/rest/client/utils/Util.java

```diff
@@ -76,17 +88,37 @@
         return map;
     }
 
+    /**
+     * Converts a {@link JsonObject} into a {@link Map}
+     * <p>
+     * The {@link Map} entries can be of types {@link Boolean}, {@link List},
+     * {@link Map}, {@link Number}, {@code null} and/or {@link String}.
+     * </p>
+     * @param jo
+     * @return 
+     */
     public static Map processJsonObject(JsonObject jo) {
```

Comment:
Maybe rename "jo" into "jsonObject" or so? The former is not really clear in what it means ("jo" here is no common abbreviation)